chapter-A.23-struct/15-struct-type-alias.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+
+type People1 struct {
+	name string
+	age int
+}
+
+type People2 struct {
```

Comment:
There is an example already for covering type alias. it is available in [chapter-A.23-struct/14-struct-alias.go](https://github.com/novalagung/dasarpemrogramangolang-example/blob/master/chapter-A.23-struct/14-struct-alias.go).

If this proposed code is trying to tell an example of type alias that value is derived from anonymous struct, then this block of code is wrong. it should be:

```go
type People2 = struct {
    name string
    age int
}
```